scripts/create-dist

```diff
@@ -39,7 +39,7 @@ fs.writeFileSync(
 );
 
 spawnSync(
-  'yarnpkg',
+  'yarn',
```

Comment:
FYI, this was `yarnpkg` instead of `yarn` because `yarn` v1 (which this repo is built to run with) had `yarnpkg` as an alias for `yarn` because `yarn` by itself clashes with the command line tool for Hadoop YARN. Apparently a similar alias doesn't exist in `yarn` v2, which is why this line caused problems (and why generally this project didn't run when used with `yarn` v2 – v1 and v2 aren't compatible).

Reply:
when I've published this I used `npm` and never had to deal with these issues but glad you now about these gotchas ... if you need an amend to this PR please file one and I'll approve ASAP 👍

Reply:
ACK. Currently working on getting the fork at https://github.com/decked/twemoji up to NPM (with working GH pages and twemoji-parser too), so I actually only pushed the fix there. Someone pointed out that it may actually be a bad idea for me to publish to the original NPM package even if I (and some previous employees who left long before Elon) have the right to.

Reply:
I'd be careful too ... here changes are fine and can be reverted, if they want, while once published on npm on behalf of the company you don't work for anymore, even if it's the company that should remove undesired / outdated publishers, might be more trouble. Use another name, maybe? That's how I'd go.

Reply:
Yeah, exactly. The plan is to namespace it under `@twemoji` as `@twemoji/api` (this package) and `@twemoji/parser` (for twemoji-parser) as two entirely new NPM packages. Twitter doesn't own any kind of trademark to the word "twemoji", so that allows it to be discoverable without publishing to the original twemoji package (even if NPM would let me). Don't want to cause unnecessary trouble, even if I believe it shouldn't be problematic in theory.

Reply:
IANAL and I believe I was the one coming up with that "*silly*" name so yours sound like a plan.
If you ever need any of my knowledge/background/advice around this fork just ping me and I'll happily answer or contribute as I can. This project has a personal history to tell and I love it very much so I'll be helping as I can whenever I can to keep it alive. Thanks for your effort indeed, it's really appreciated!